modules/identity-provider-add.adoc

```diff
@@ -37,15 +48,46 @@ $ oc apply -f </path/to/CR>
 If a CR does not exist, `oc apply` creates a new CR and might trigger the following warning: `Warning: oc apply should be used on resources created by either oc create --save-config or oc apply`. In this case you can safely ignore this warning.
 ====
 
+ifndef::no-username-password-login[]
 . Log in to the cluster as a user from your identity provider, entering the
 password when prompted.
 +
 ----
 $ oc login -u <username>
 ----
+endif::no-username-password-login[]
+
+ifdef::no-username-password-login[]
+
+. Obtain a token from the OAuth server.
++
+As long as the `kubeadmin` user has been removed, the `oc login` command provides instructions on how to access the `/oauth/token/request` token request endpoint, where you can log in and retrieve the token.
```

Comment:
I would just say "... instructions on how to access a web page where the token can be retrieved."

If you want, I can set up a cluster with github IdP so that you can get the picture about what's actually going on.

Reply:
Thanks, I've updated. And appreciate your help with testing, I've also added how to get to the token request page from the console.